scripts/asm-installer/install_asm

```diff
@@ -602,6 +603,7 @@ OPTIONS:
   --root_cert            <FILE PATH>
   --cert_chain           <FILE PATH>
   --ca_name              <CA NAME>
+  --revision             <CUSTOM_REVISION>
```

Comment:
Let's use `--revision_name`, change `CUSTOM_REVISION` to `REVISION_NAME`, and add a short flag `-r`. The first two are to align with later usage in other places in ASM, the last one because I imagine this is a flag that will get specified a lot.